bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/regexp/TreeBuilder.java

```diff
@@ -35,14 +35,17 @@
 import io.ballerina.runtime.internal.values.RegExpValue;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 /**
  * Regular expression tree builder for Ballerina regular expression node.
  *
  * @since 2201.3.0
  */
 public class TreeBuilder {
+
```

Comment:
Please refer to https://github.com/ballerina-platform/ballerina-lang/pull/39727#discussion_r1134956518